### ISSUE TITLE
[1.18.2] Crash At Startup

### ISSUE THREAD

**Issue by DeshiDesu** [TheHamester/HitIndicator#3]
hitindication-1.18.2-1.2.jar

[crash-2023-05-28_01.44.17-fml.txt](https://github.com/TheHamester/HitIndicator/files/11587009/crash-2023-05-28_01.44.17-fml.txt)

**Comment by TheHamester** [comment]
```
Exception message: org.spongepowered.asm.mixin.injection.throwables.InjectionError: Critical injection failure: Redirector dcfixes$disableDebug(Ljava/io/PrintStream;Ljava/lang/Object;)V in dawncraft-fixes.mixins.json:hitindication.MixinHitEvents failed injection check, (0/1) succeeded. Scanned 1 target(s). Using refmap dawncraft-fixes.refmap.json
```

theres no longer need for the mixin into this function, since i removed println that caused the log spam in the previous version.

**Comment by TheHamester** [comment]
im refering to this issue https://github.com/TheHamester/HitIndicator/issues/2
and this tweak https://github.com/SettingDust/dawncraft-fixes/commit/28cf211f80773cf3ea25d90829d5f66e58b82b2b

**Comment by TheHamester** [comment]
if you are part of DawnCraft development team, please inform SettingDust about this
if you are just adding a newer version of my mod to dawncraft to play with it, this is the thing that causes the issue.

**Comment by SettingDust** [comment]
https://www.curseforge.com/minecraft/mc-mods/mod-compat-fixes/files/4559155
